src/servers/grpc_server_v2.cc

```diff
@@ -336,12 +336,17 @@ class HandlerState {
     context_ = context;
     unique_id_ = RequestStatusUtil::NextUniqueRequestId();
     step_ = start_step;
+    sequence_id_ = 0;
```

Comment:
I don't think the GRPC endpoint needs to be aware of the sequence ID. It will be in the request and response objects and completely managed by Triton core.